packages/plugin-wdm/test/unit/spec/device.js

```diff
@@ -44,6 +44,10 @@ describe(`plugin-wdm`, () => {
       });
     });
 
+    describe(`#determineService()`, () => {
+      it(`determines the service backing the specified url`, () => assert.becomes(device.determineService(`https://conv-a.wbx2.com/conversation/api/v1/conversations`, `conversation`)));
```

Comment:
```js
assert.becomes(device.determineService(`https://conv-a.wbx2.com/conversation/api/v1/conversations`, `conversation`)));
```

Seems like it shouldn't work because we are providing two params to `determineService()` and only one to `assert.becomes()`. This means that the test is asserting that determineService(`https://conv-a.wbx2.com/conversation/api/v1/conversations`, `conversation`) becomes `undefined`.  I think what we want is this, right?

```js
assert.becomes(device.determineService(`https://conv-a.wbx2.com/conversation/api/v1/conversations`), `conversation`));
```